### ISSUE TITLE
Original firmware file?

### ISSUE THREAD

**Issue by patchy8736** [mooninite/g20s-pro-bt#1]
Hi,

is there any chance that you also got the original firmware file for reverting back?
Since the upgrade the G20s Pro very often crashes. When this happens it stops reacting to button inputs, after a few seconds the blue backlight turns off and won't turn on again. You have to remove the batteries and put them back in to "reboot" the remote.
This never happened before with the old firmware.

Also with the new firmware the remote starts in "air mouse mode" when being woken up and you have to turn the mouse off every time. This is the opposite to the old firmware. With the old firmware you had to turn air mouse on via button if you wanted to use it.

TLDR: I need to revert back to the original firmware pls. :(

**Comment by mooninite** [comment]
I do not have the "original" version. I have not experienced the issues you described.

**Comment by patchy8736** [comment]
That‘s too bad.
Would you be able to send contact data from the person that sent you the firmware file?
Cheers

**Comment by mooninite** [comment]
I have it, but I would not want to publicly share it out of courtesy. Is there an email address you could post here? I tried looking at your profile and didn't find one.

**Comment by patchy8736** [comment]
Sure, you can use: 00_softdrink.komet@icloud.com
Thanks in advance!

**Comment by patchy8736** [comment]
Thanks for your help. No response yet.
I think I gonna use this opportunity and look for a new remote. :P

**Comment by Hall0wMan31** [comment]
Can i get the original file too?

**Comment by Hall0wMan31** [comment]
My email is: egyedifiok@gmail.com
